test/recipes/80-test_cms.t

```diff
@@ -1154,3 +1154,11 @@ with({ exit_checker => sub { return shift == 3; } },
             "issue#21986");
         }
     });
+
+# Test for problem reported in #22225
+ok(!run(app(['openssl', 'cms', '-encrypt',
```

Comment:
I'd suggest using the exit_checker as in above test cases as otherwise crash or any other unexpected failure can make this test pass.